log/logger.go

```diff
@@ -62,7 +62,7 @@ func NewLoggerWithKV(key, value string) Logger {
 	return zeroLogWrapper{&logger}
 }
 
-// NewCustomLogger returns a new logger with the given zerolog logger
+// NewCustomLogger returns a new logger with the given zerolog logger.
```

Comment:
@julienrbrt godoc should be full, punctuated sentences.



When they are rendered on pkg.go.dev without punctuation, it looks sloppy, and it's not always clear if they are finished sentences.

<img width="562" alt="Screen Shot 2023-03-03 at 9 14 11 AM" src="https://user-images.githubusercontent.com/290647/222742754-28f2a152-0caa-4b42-99c4-685dfe0293a7.png">

Reply:
Sure thing! I'll watch out, it does look sloppy without indeed 👍🏾

Reply:
this is good to know, i bet there are many other areas we do this